go.sum

```diff
@@ -770,6 +770,8 @@ github.com/stretchr/testify v1.3.0/go.mod h1:M5WIy9Dh21IEIfnGCwXGc5bZfKNJtfHm1UV
 github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81PSLYec5m4=
```

Comment:
Based on the go.mod changes, I'd have expected some lines to be removed here.

Reply:
I have removed go.sum, removed my local go cache, and run go mod tidy, I can't do better than my last iteration.